extensions/stackdriver/log/exporter.cc

```diff
@@ -81,10 +89,15 @@ void ExporterImpl::exportLogs(
     bool is_on_done) {
   is_on_done_ = is_on_done;
   for (const auto& req : requests) {
-    context_->grpcSimpleCall(grpc_service_string_, kGoogleLoggingService,
-                             kGoogleWriteLogEntriesMethod, *req,
-                             kDefaultTimeoutMillisecond, success_callback_,
-                             failure_callback_);
+    auto result = context_->grpcSimpleCall(
+        grpc_service_string_, kGoogleLoggingService,
+        kGoogleWriteLogEntriesMethod, *req, kDefaultTimeoutMillisecond,
+        success_callback_, failure_callback_);
+    if (result != WasmResult::Ok) {
+      LOG_WARN("failed to make stackdriver logging export call");
+      break;
```

Comment:
It is not clear to me if we should use break or continue. Both are good in terms of the counter. I will let you make the final call.

Reply:
grpcSimpleCall returns non-ok status if there is something bad in Grpc service configuration, which is unlikely going to work for the following requests.